packages/react/runtime/__test__/list.test.jsx

```diff
@@ -1952,9 +1952,9 @@ describe('list-item JSXSpread', () => {
       b.insertBefore(d2);
       b.insertBefore(d3);
 
-      d1.setAttribute(0, { 'item-key': '1', 'full-span': true });
-      d2.setAttribute(0, { 'item-key': '2', 'full-span': true });
-      d3.setAttribute(0, { 'item-key': '3', 'full-span': true });
+      d1.setAttribute(0, { 'item-key': '1', 'full-span': true, 'recyclable': true });
```

Comment:
Looks like the test is added only in "list-item JSXSpread", could u pls add some without using JSXSpread?